src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -86,11 +93,16 @@ public void deleteMessage(String ID) {
    */
   public List<Message> getMessages(String recipient) {
```

Comment:
Do you mind updating the javadoc comment for this method? It looks like it hasn't been changed to reflect the user -> recipient change, and you can also include the fact that this now only includes public messages.

Reply:
done